src/styles/index.ts

```diff
@@ -4045,6 +4045,20 @@ const styles = (theme: ThemeColors) =>
             width: 1,
         },
 
+        emptySageIntacctUserDimensionsSubtitle: {
```

Comment:
This all feels like super specific styles. Why do we need them? Can you show me a screenshot of what they do exactly?

Reply:
You're right, these two styles seem too specific. I extracted a more universal style and just added two attributes that I needed(textAlignCenter and textAlignLeft) right in the components. WDYT

Reply:
Love it, thanks for cleaning that up! Will check out the new commit.